packages/data-transport-layer/src/services/server/service.ts

```diff
@@ -51,12 +51,23 @@ const optionSettings = {
       return validators.isUrl(val) || validators.isJsonRpcProvider(val)
     },
   },
+  l2RpcProvider: {
+    validate: (val: any) => {
```

Comment:
I think you can / should use `unknown` here as opposed to `any`. Also, the `isUrl` function can / should have a signature like `isUrl(val: unknown): val is URL` (or, `string` instead of `URL` since we don't have a `URL` type)

Reply:
I didn't know that the `unknown` type existed but I agree that it makes sense to use here. Updating `isUrl` is out of scope of this PR